### ISSUE TITLE
Adguard browser extension breaking some websites in Edge browser

### ISSUE THREAD

**Issue by GauravSahare27** [AdguardTeam/AdguardBrowserExtension#1661]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ ] I am running Adguard Adblocker v3.4.31 (downloaded from official website of Microsoft Edge extension page)
(allow access to file URLs also enabled)

### Issue Details

- **AdGuard version:**
  - 
- **Browser and version:**
  - 
- **Operating system and version:**
  - 
- **Filters you use in AdGuard:**
  - 
- **Other browser extensions:**
  - 

### Actual Behavior

</details>

**Comment by adgkart** [comment]
Hi! Could you please provide us more details about AdGuard version, Browser and version, Operating system and version, Filters you use in AdGuard, and what websites were broken?

**Comment by GauravSahare27** [comment]
1. Adguard Adblocker browser extension v3.4.34 (downloaded from Microsoft Edge extension page).
2. Microsoft Edge(Chromium) Stable Build v83.0.478.58
3. Windows 10 Home Edition 64 Bit Build 19041 (May Update 2020).
List of Enabled Filters
(i) Ad Blocking > Adguard Base Filter.
(ii) Privacy > Adguard Tracking Protection Filter.
(iii) Annoyances > Adguard Annoyances Filter.
(iv) Security > Malware Domains Blocklist, Spam404, NoCoin Filter List.

Broken websites:
https://www.microsoft.com/en-us/edge
https://www.microsoft.com/en-us/edge/business/download

**Comment by adgkart** [comment]
Hi! Thank you for your reply! You also wrote that "extension breaking websites in Edge", what do you mean by "breaking"? Could you please send us screenshots and check this issue in incognito mode?

**Comment by GauravSahare27** [comment]
Hello sir,

Actually I was facing site breaking issue in previous version of Microsoft Edge browser browser, but after update to version 83.0.478.58 site breaking problem gone, now website is opening, but not properly.
But yes in Incognito Mode, website is opening properly because no addon is enabled.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by GauravSahare27** [comment]
But I am surprised that this problem not happening with other browsers when I open same URL, even with addons are enabled.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by GauravSahare27** [comment]
Hello sir,
I want to inform that the issue I reported has been fixed in updated version 83.0.478.61 by Microsoft.
That means this problem is not generated by Adguard.
There was some technical issues that addressed later by Microsoft and now fixed.
I request to remove this issue from section of issue.
In previous browser versions, these websites was open properly in InPrivate Tab due to technical issue on that sites.
But now this issue has fixed.
